gui/src-tauri/src/main.rs

```diff
@@ -286,3 +299,15 @@ fn valid_java_paths() -> Vec<(OsString, i32)> {
 		})
 		.collect()
 }
+
+fn executable<W: AsRef<OsStr> + ?Sized>(bin: &W) -> OsString {
+	let bin = bin.as_ref();
+	if cfg!(windows) {
+		let mut new = OsString::with_capacity(bin.len() + 4);
```

Comment:
We are working in a codebase that doesn't have rust developers, please focus on keeping the code as simple and readable as possible. just do `OsString::new()` - reserving a particular capacity is not necessary here

Reply:
i would do this even in js and it would be less unreadable, ``OsString::with_capacity`` is very verbose tbh